superdesk/newsroom-core
# [CPCN-177] - Add `Source` column to Content Activity report

### Purpose
This PR introduces a new column to the Content Activity report.

### What has changed
- Added a new column (`source`) to the report both UI and CSV export
- Removed the usage of `lodash.get` from the `ContentActivity.tsx` component
- Completed typings for the properties of the component
- Remove the usage of `PropTypes` in favour of Typescript interfaces
- Little code clean up 

### Steps to test
- Checkout the branch
- Run the project and go to Reports > Content Activity
- Run the report
- Notice that the column has been added and everything else is working as expected

### Checklist

Thanks for reviewing! 

Resolves: [CPCN-177]

[CPCN-177]: https://sofab.atlassian.net/browse/CPCN-177?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
